snmp/datadog_checks/snmp/data/profiles/cisco-catalyst.yaml

```diff
@@ -372,5 +372,11 @@ sysobjectid:
   - 1.3.6.1.4.1.9.1.2498
   - 1.3.6.1.4.1.9.1.2501 # ciscoC9410R
   - 1.3.6.1.4.1.9.1.2648 # ciscoC9606R
+  - 1.3.6.1.4.1.9.1.2688 # ciscoCat9500VirtualStack
   - 1.3.6.1.4.1.9.1.2694 # cisco9200FixedSwitchStack
   - 1.3.6.1.4.1.9.1.2695 # cisco9200RFixedSwitchStack
+  - 1.3.6.1.4.1.9.1.2819 # ciscoC850012X
+  - 1.3.6.1.4.1.9.1.2961 # ciscoC82001N4T
+  - 1.3.6.1.4.1.9.1.2963 # ciscoC83002N2S4T2X
+  - 1.3.6.1.4.1.9.1.3004 # ciscoC8000V
+  - 1.3.6.1.4.1.9.1.3069 # ciscoC8500L8S4X
```

Comment:
Could you add to the PR description (e.g. link to sources) how you verified those new devices are Cisco Catalyst devices? :)

Reply:
Hi @AlexandreYang ! I added links to sources in my description.